audit/projects/kubernetes-public/iam.json

```diff
@@ -117,7 +117,8 @@
       "members": [
         "group:gke-security-groups@kubernetes.io",
         "serviceAccount:gke-nodes-aaa@kubernetes-public.iam.gserviceaccount.com",
-        "serviceAccount:k8s-infra-monitoring-viewer@kubernetes-public.iam.gserviceaccount.com"
+        "serviceAccount:k8s-infra-monitoring-viewer@kubernetes-public.iam.gserviceaccount.com",
+        "user:tjferrara@google.com"
```

Comment:
This was me, https://github.com/kubernetes/k8s.io/issues/2364#issuecomment-882764991